desc/objectives/_equilibrium.py

```diff
@@ -15,14 +15,23 @@
 
 
 class ForceBalance(_Objective):
-    """Radial and helical MHD force balance.
+    r"""Radial and helical MHD force balance.
 
-    F_rho = sqrt(g) (B^zeta J^theta - B^theta J^zeta) - grad(p)
-    f_rho = F_rho |grad(rho)| dV  (N)
+    Given force densities:
 
-    F_helical = sqrt(g) J^rho
-    e^helical = -B^zeta grad(theta) + B^theta grad(zeta)
-    f_helical = F_helical |e^helical| dV  (N)
+    Fᵨ = √g (B^ζ J^θ - B^θ J^ζ) - ∇ p
```

Comment:
woah how did you make this format like that with the special characters?

Reply:
i think you can just copy paste the output of latex since it's unicode

Reply:
Yeah just Unicode. Figured thats the best option for having both the raw text and html docs be readable